tests/test-cases/Group6-VIC-Machine/6-16-Config.robot

```diff
@@ -148,3 +148,12 @@ Configure VCH https-proxy through vch id
     ${rc}  ${output}=  Run And Return Rc And Output  govc vm.info -e %{VCH-NAME} | grep HTTPS_PROXY
     Should Be Equal As Integers  ${rc}  1
     Should Not Contain  ${output}  proxy.vmware.com:3128
+
+Configure VCH DNS server
+    ${output}=  Run  bin/vic-machine-linux inspect --name=%{VCH-NAME} --target=%{TEST_URL} --thumbprint=%{TEST_THUMBPRINT} --user=%{TEST_USERNAME} --password=%{TEST_PASSWORD} --timeout %{TEST_TIMEOUT} --conf
```

Comment:
If https://github.com/vmware/vic/pull/5418 gets merged before this, please change `--conf` to `config`.